cmd/maestroctl/cmd/plan/helpers.go

```diff
@@ -0,0 +1,35 @@
+package plan
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+const (
+	defaultConfigPath = ".kube/config"
+)
+
+var (
+	instance   string
+	kubeConfig string
+	namespace  string
+)
+
+// mustKubeConfig checks if the kubeconfig file exists.
+func mustKubeConfig() {
```

Comment:
Should this exit the application with the error message, or just write it to the console like it does?

Reply:
Yep, you`re right and exiting with `os.Exit(1)` seems like the right choice here.

Reply:
Did you add this?

Reply:
I thought I did: https://github.com/maestrosdk/maestro/pull/59/commits/b1a42b2d76195095e0a9b656edf47e074468d582#diff-ca52abd2894fe4a50e247880a137ba9eR33

Instead of adding a line under the `log.Printf` statement I decided to use `log.Fatalf` as it does `os.Exit(1)`